api/src/main/java/com/okta/sdk/client/ClientBuilder.java

```diff
@@ -325,13 +329,62 @@
      * of relying on the default location + override/fallback behavior defined
      * in the {@link ClientBuilder documentation above}.
      *
-     * @param privateKey the fully qualified string path to the private key (PEM file).
+     * @param privateKey either the fully qualified string path to the private key PEM file (or)
+     *                   the full PEM payload content.
      * @return the ClientBuilder instance for method chaining.
      *
      * @since 1.6.0
      */
     ClientBuilder setPrivateKey(String privateKey);
 
+    /**
+     * Allows specifying the private key (PEM file) file object (for private key jwt authentication) directly instead
+     * of relying on the default location + override/fallback behavior defined
+     * in the {@link ClientBuilder documentation above}.
+     *
+     * @param privateKeyFile representing the private key (PEM file) file object.
+     * @return the ClientBuilder instance for method chaining.
+     *
+     * @since 3.0.0
+     */
+    ClientBuilder setPrivateKey(File privateKeyFile);
```

Comment:
This overloaded method is not needed in my opinion (`setPrivateKey(Path privateKeyPath)` can be used instead).

Reply:
fixed